hail/python/hail/docs/_static/annotationdb/annotationdb.js

```diff
@@ -0,0 +1,79 @@
+
+
+//CheckAll functionality for UI
+$("#checkAll").click(function(){
+    $('input:checkbox').not(this).prop('checked', this.checked);
+});
+
+//Get and append JSON information to Table
+$.getJSON("https://api.myjson.com/bins/1fn7f7",
+                function (data) {
+                    var tr = data.report
+                    for (var i = 0; i < data.length; i++) {
+                        tr = $('<tr/>');
+                        tr.append("<td><input type='checkbox' class='checkboxadd' value='"+data[i].name+"' onClick='updateTextArea()'/>&nbsp;</td>")
+                        tr.append("<td>" + data[i].name + "</td>");
+                        tr.append("<td>" + data[i].version + "</td>");
+                        tr.append("<td>" + data[i].n_rows + "</td>");
+                        tr.append("<td>" + data[i].n_cols + "</td>");
+                        $('.table1').append(tr);
+                    }
+                });
+
+//Allows filtering of table
+function searchTable() {
```

Comment:
We should change the name of this function to better reflect its purpose: `filterTable` and then we can remove the comment.